src/modules/drive/AddMenu/AddMenuContent.jsx

```diff
@@ -63,6 +64,9 @@ const AddMenuContent = forwardRef(
         {canCreateFolder && !isEncryptedFolder && (
           <AddFolderItem onClick={onClick} isReadOnly={isReadOnly} />
         )}
+        {canCreateFolder && !isPublic && flag('drive.shared-drive.enabled') && (
```

Comment:
nit: we probably should refactor this and use actions with ActionsMenu instead of "manual" ActionsMenuItems 🤔